jobs/service-fabrik-broker/spec

```diff
@@ -181,6 +189,15 @@ properties:
   internal.domain_socket.path:
     description: "domain socket path used to communicate with broker monitoring agent"
     default: /tmp/sfevents
+  internal.floating_ip:
+    description: "Floating IP used for Ip re-assignment in HA setup"
+    default: 10.11.252.10
+  internal.master_ip:
+    description: "Ip used for master vm in HA setup"
+    default: 10.11.252.11
+  internal.slave_ip:
```

Comment:
What happens for a non-HA setup? I hope these are not mandatory. If they are not, then we should as well move them to keepalived job.

Reply:
+1